### PR TITLE
[JENKINS-39547] Reproduce jar cache corruption in test

### PR DESCRIPTION
**Pull request by olivergondza**
[JENKINS-39547](https://issues.jenkins-ci.org/browse/JENKINS-39547)

### CODE REVIEW

**Comment by olivergondza** [comment]
Actual fix in remoting: https://github.com/jenkinsci/remoting/pull/130/files

File: test/src/test/java/hudson/model/Slave2Test.java [hunk 1]
```diff
@@ -118,6 +132,45 @@ private void assertJnlpJarUrlIsAllowed(@Nonnull Slave slave, @Nonnull String url
     }
 
     @Test
+    public void corruptSlaveJar() throws Exception {
```
Comment (oleg-nenashev) [inline test/src/test/java/hudson/model/Slave2Test.java]: Please add `@Issue`

**Review by oleg-nenashev — APPROVED** [review]
The test looks good to me. Sorry for the review delay in the remoting PR

**Comment by daniel-beck** [comment]
@olivergondza What's the nature of work-in-progress here?

**Comment by olivergondza** [comment]
The test is failing until remoting 3.3 is merged.

**Comment by oleg-nenashev** [comment]
should be passing now

**Comment by olivergondza** [comment]
I am keeping the WIP as I encountered test failures.

**Comment by oleg-nenashev** [comment]
@olivergondza Is there a chance we need to rollback remoting-3.3 due to these failures? Or is it just a test case error?

**Comment by olivergondza** [comment]
The problem I expected to resolve have not disappeared: https://jenkins.ci.cloudbees.com/job/core/job/jenkins-core/6688/testReport/junit/hudson.model/Slave2Test/corruptSlaveJar/. Likely a test case error or the fix is not working - should not be a reason to revert.

**Comment by olivergondza** [comment]
The test does not reproduce the issue. Closing

**Pull request closed without merging**